service/controller/v12/resource/s3object/desired_test.go

```diff
@@ -24,21 +26,23 @@ func Test_DesiredState(t *testing.T) {
 		},
 	}
 
+	masterKeyPattern := "cloudconfig/v[\\d_]+/master"
+	workerKeyPattern := "cloudconfig/v[\\d_]+/worker"
```

Comment:
I refactored the test to match on a regex so we only need to update the constant in the cloudconfig service.